app/services/bulkrax/remove_relationships_from_importer.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+module Bulkrax
+  # This module is rather destructive; it will break relationships between the works, file sets, and
```

Comment:
Technically, this doesn't break relationships for file sets (see line 55): 

```ruby
next if obj.is_a?(FileSet) # FileSets must be attached to a Work
```

Whether we _should_ break file set relationships, I'm open to discussion on.

Reply:
I am amending the inline docs to remove the FileSet reference.